test-support/helpers/ember-i18n/test-helpers.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+// example usage: find(`.header:contains(${t('welcome_message')})`)
+Ember.Test.registerHelper('t', function(app, key){
+  var i18n = app.__container__.lookup('service:i18n');
+  return i18n.t(key);
+});
+
+// example usage: expectTranslation('.header', 'welcome_message');
+Ember.Test.registerHelper('expectTranslation', function(app, element, key){
+  var i18n = app.__container__.lookup('service:i18n'),
+      translation = i18n.t(key);
+
+  return ok(find(`${element}:contains(${translation})`),  `Found translation key ${key} in ${element}`);
```

Comment:
@rwjblue you work on [ember-mocha](https://github.com/switchfly/ember-mocha). Do you have any suggestions for how to write this in a test-framework-agnostic way?